src/components/app/list-new/HelmAppList.tsx

```diff
@@ -357,7 +370,68 @@ export default function HelmAppList({
                     <span className="app-list__cell-header">Last deployed at</span>
                 </div>
             </div>
-        );
+        )
+    }
+
+    const renderFetchError = (externalHelmListFetchError: string, index: number) => {
+        return (
+            <div className="bcn-0" key={index}>
+                <div className="h-8"></div>
+                <div className="ea-fetch-error-message above-header-message flex left">
+                    <span className="mr-8 flex">
+                        <ErrorExclamationIcon className="icon-dim-20" />
+                    </span>
+                    <span>{externalHelmListFetchError}</span>
+                    <CloseIcon
+                        className="icon-dim-24 dc__align-right cursor"
+                        onClick={() => _removeExternalAppFetchError(index)}
```

Comment:
Extract this inline function

Reply:
Done with the changes. Using dataset attr for getting index value.